### ISSUE TITLE
Code of conduct for donations to the Giveth DAC & access to the Giveth DApp

### ISSUE THREAD

**Issue by krrisis** [Giveth/Communication#71]
We need to work out a procedure 

1) to evaluate each project/org that contacts us and wants to donate to Giveth (or to future projects on our DApp)? 

- what are the questions we should ask
- what are the projects to which we say no for sure and how do we explain this
- who is our SPOC for this and who follows up for which aspect

2) what guidance can we give them to talk about this collaboration & how will we support their efforts 
- when do we get them on the DApp as a DAC or a Campaign and when do we just propose to use some code and send part of their fee/proceeds to our multisig? 
- what wording do we use (partnership? dac? ...) and how do we support them communication-wise?

**Comment by krrisis** [comment]
@vojtechsimetka wrote out a great start to this thought process, after a previous loomio discussion, see below: 

a) First one defining 3 groups of donations:

Safe donations
To be cautious about
Unsafe

b) Deciding what to do for each category of the above
Ideas:

Part of all money of donation group can go to a hold contract to be used for legal liability
There could be a delay mechanism that holds the money for 1 month before we can use them
Returning money to the victims of scam
Using the donations from illicit activity to support fight against the activity
How to we deal with greenwashing?

c) Assigning types of the donations into the donation groups
Taking the categories from the original thread and assigning them to the 3 donation groups https://www.loomio.org/d/iK2aVg05/what-donations-do-we-accept-and-what-do-we-do-with-those-we-decide-to-reject-

d) Defining a process of categorising donation
Anyone from Giveth should be able to trigger voting on any of the donations we received. The outcome of the vote should be assigning the donation to one of the 3 donation groups and following the process for that group.

(link to text in our riot - closed group, just for reference [here](https://matrix.to/#/!VUpIHOCgPdHvUrtwXq:matrix.org/$1530693393906492tZsqo:matrix.org) )

**Comment by Danibelle** [comment]
It appears this is written specific to Givers; assuming it correlates to the onboarding process for Makers, which I haven't yet seen.

The open source regenerative community project I referenced on the 8/8 comm call is Bloomnetwork.org and this document for engagement (in any direction - using term Stakeholders as umbrella) is the one I spoke of... 

[Engagement Policy](https://docs.google.com/document/d/1T2u4Q7x7U-8q_SS0gmQVo0GCBnic5WqZJF1aFIjJt5I/edit?usp=sharing)

**Page 2 has 3 tiers that correspond to those @vojtechsimetka established previously and shown above.
So, on the same 'page' around this - next question is how to determine which tier an entity is currently in - and does Giveth consider whether collaborating would help the entity aid in transitioning 'up'?
**Page 6 lays out a procedure for the screening policy with defined positive/negative indicators for each of the 7 areas of cultural transformation that make up the networks goals.

**Comment by Danibelle** [comment]
Moving this to In Progress, I am thinking an incremental solution as we onboard.

Beginning simply with a few screening questions as part of the Beta Interest Questionnaire to be distributed for pre-qualifying our next collaborators, as well as having acceptance of funds or access to the DApp linked to an affirmation of agreement to the Giveth mission and values.

**Comment by krrisis** [comment]
To me this is a practical thing on which the governance circle could help, possibly the gov-lab initiative @geleeroyale ? This is a real governance point for which we definitely need some brainstorming, experimentation & documentation.

**Comment by krrisis** [comment]
little reminder for this one - is this gov-lab territory @Joshfairhead @geleeroyale ? wdyt?

**Comment by geleeroyale** [comment]
Thanks for the ping @krrisis. I don't think this is specifically for gov-lab, as I see that more as R&D, but I agree that we should discuss this in governance circle. We have some good voices here in this thread that lay a foundation for a protocol we could - as a community - define further. 

Because Giveth is a platform for delegating donations, I recommend we primarily define the edge cases (funds we don't want to accept at all), actions we can take if there still is a donation from an unwanted source and put it into the perspective of dealing with a pseudonymous system where it is usually hard to impossible to identify the owner of a donation.

**Comment by Danibelle** [comment]
I suggest a simple disclaimer where ever our donation addresses are listed that says something to the effect of:
If you would like to use the Giveth name in publishing your charitable donation or otherwise support and collaborate with us publicly, please let us know in advance by contacting XXX.
In the event the Giveth brand is represented as a partner without our prior knowledge and agreement, we may take action up to refunding donations and issuing a statement of non-cooperation.

**Comment by krrisis** [comment]
Yeah I think that is definitely a quick win! could we add that @geleeroyale ? feel free to put my name there '..by contacting @krrisis in our [riot chat](http://giveth.io/join)' 

That is one step for the donations, the question on who we refuse on the dapp is another one. And so is actually whose donations we accept. Personally I stay open to accepting all donations, we use it for good :)

**Comment by Joshfairhead** [comment]
I'm for Dani's suggestion. Looks like theres agreement. Assuming it'll go
up for a loomio and then be added to accepted proposals if passed?

---
Mobile: +44 (0) 7412090234
Calendar of my non-liquid commitments
<https://calendar.google.com/calendar/embed?src=joshafairhead%40gmail.com&ctz=Europe%2FLondon>
Calendar of my liquid arrangements
<https://calendar.google.com/calendar/embed?src=mef94j8i5evn7f89p76bsql89s%40group.calendar.google.com&ctz=Europe%2FLondon>
Loomio <https://www.loomio.org/g/Yx3rGU5I/techtonic>

ᐧ

**Comment by geleeroyale** [comment]
Yea - let's close the chapter with a loomio and crack the champagne ... I mean the sparkling water! :champagne: 

I find the statement on what constitutes a valid DAC for us even more important, but that should be its own discussion.

**Comment by Danibelle** [comment]
Following up on our 'Ethics of the DApp' session at the BCN Retreat; I recommend we consider forking the Community Guidelines and Code of Conduct shared to us by RadicalxChange and attached here. 
 
[Code of Conduct (1).pdf](https://github.com/Giveth/Communication/files/3307119/Code.of.Conduct.1.pdf)
[RadicalxChange - Chapter Guidelines.pdf](https://github.com/Giveth/Communication/files/3307120/RadicalxChange.-.Chapter.Guidelines.pdf)

Code of Conduct for RxC was forked and modified from here:
http://citizencodeofconduct.org/

**Comment by geleeroyale** [comment]
@Danibelle We do have a code of conduct for the **community**! (https://wiki.giveth.io/governance/codeofconduct/)

Please feel free to propose changes to that!

---
Both the one we use and the RadXChange one seem to be geared towards behavior in a community, whereas this discussion is about **donations**.

I don't think those are even similar, so we will have to do the work and make completely new ones for:

- Donations to Giveth DAC
- Access to the Giveth DApp

I am highly in favor of using the work we did on the retreat however - at this point it comes down to writing the thing.

**Comment by loietaylor** [comment]
Hmm, I would like to weigh in here to affirm, that this Code of Conduct for behavior and acceptance/privileges in the giveth community is very different from the Ethics of the DApp conversation and what walls do we have facing incoming donations... I totally glazed over the fact that you were connecting those two @danibelle in the loomio and was just all for supporting real focus on our healthy community standards. Both are necessary and probably should be cornered to their zone so as not to take energy away from one another!
